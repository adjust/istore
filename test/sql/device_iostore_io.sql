BEGIN;
CREATE TABLE device_istore_io AS SELECT 'bot=>1,mac=>2'::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT '"bot"=>"1","bot"=>"2"'::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT '"bot"=>"1","mac"=>"2"'::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT '"mac"=>"+1","bot"=>"2"'::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT ' "mac"=>"+1","bot"=>"2"'::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT '"mac"=> "+1","bot"=>"2"'::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT '"mac"=>"+1" ,"bot"=>"2"'::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT '"mac"=>"+1", "bot"=>"2"'::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT '"mac"=>"+1","bot"=>"2" '::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE device_istore_io AS SELECT '"mac"=>"+1","bot"=>"2" '::device_istore;
SELECT * FROM device_istore_io;
ROLLBACK;
