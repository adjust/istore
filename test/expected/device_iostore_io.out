BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT 'bot=>1,mac=>2'::device_istore;
SELECT * FROM device_iostore;
     device_istore     
-----------------------
 "bot"=>"1","mac"=>"2"
(1 row)

ROLLBACK;
BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT '"bot"=>"1","bot"=>"2"'::device_istore;
SELECT * FROM device_iostore;
 device_istore 
---------------
 "bot"=>"3"
(1 row)

ROLLBACK;
BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT '"bot"=>"1","mac"=>"2"'::device_istore;
SELECT * FROM device_iostore;
     device_istore     
-----------------------
 "bot"=>"1","mac"=>"2"
(1 row)

ROLLBACK;
BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT '"mac"=>"+1","bot"=>"2"'::device_istore;
SELECT * FROM device_iostore;
     device_istore     
-----------------------
 "bot"=>"2","mac"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT ' "mac"=>"+1","bot"=>"2"'::device_istore;
SELECT * FROM device_iostore;
     device_istore     
-----------------------
 "bot"=>"2","mac"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT '"mac"=> "+1","bot"=>"2"'::device_istore;
SELECT * FROM device_iostore;
     device_istore     
-----------------------
 "bot"=>"2","mac"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT '"mac"=>"+1" ,"bot"=>"2"'::device_istore;
SELECT * FROM device_iostore;
     device_istore     
-----------------------
 "bot"=>"2","mac"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT '"mac"=>"+1", "bot"=>"2"'::device_istore;
SELECT * FROM device_iostore;
     device_istore     
-----------------------
 "bot"=>"2","mac"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- device_iostore_io should persist device_iostore;
-- ./spec/device_iostore_io_spec.rb:20;
CREATE EXTENSION istore;
CREATE TABLE device_iostore AS SELECT '"mac"=>"+1","bot"=>"2" '::device_istore;
SELECT * FROM device_iostore;
     device_istore     
-----------------------
 "bot"=>"2","mac"=>"1"
(1 row)

ROLLBACK;
